packages/frontend-shared/cypress/e2e/e2ePluginSetup.ts

```diff
@@ -32,6 +32,7 @@ import sinon from 'sinon'
 import fs from 'fs-extra'
 
 import { CloudQuery } from '@packages/graphql/test/stubCloudTypes'
+import { CYPRESS_REMOTE_MANIFEST_URL, NPM_CYPRESS_REGISTRY_URL } from '@packages/types/src/constants'
```

Comment:
This import is causing test failures... not sure why 🤔

Reply:
I wonder if you need to add `"@packages/types"` as a dependency to `packages/app/package.json`.

Reply:
Turns out it wasn't the import in this file that was causing the issue... see my comment [here](https://github.com/cypress-io/cypress/pull/26105#issuecomment-1470637793)